src/style.css

```
@@ -1 +1,44 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.main {
+  max-width: 96px;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--1 :nth-child(-n + 1) {
```

```suggestion
.stars--1 :nth-child(-n + 1),
... ,
.stars--5 :nth-child(-n + 5) {
```
Consider using commas to group selectors